MarsAutomation/Pages/LaguangePage.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection.Emit;
+using System.Text;
+using System.Threading.Tasks;
+using MarsAutomation.Utilities;
+using Microsoft.Extensions.DependencyModel;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.DevTools.V121.Debugger;
+using OpenQA.Selenium.Support.UI;
+
+namespace MarsAutomation.Pages
+{
+    public class LaguangePage
+    {
+
+        // Add new language record
+        public void addnewlanguage(IWebDriver driver, string languageName, int level)
```

Comment:
do not pass the driver in action methods, please pass this via the Constructor LaguangePage(Driver driver) or create a CommonDriver class and extend to it